optuna/integration/allennlp/_pruner.py

```diff
@@ -129,7 +126,7 @@ class AllenNLPPruningCallback(TrainerCallback):
 
     .. note::
         Currently, build-in pruners are supported except for
-        :class:`~optuna.pruners.PatientPruner`.
+        :class:`~pruners.PatientPruner`.
```

Comment:
The `PatientPruner` is not linked correctly. Could you revert the change?

![image](https://user-images.githubusercontent.com/3255979/143238515-c8d244ce-8820-4e61-8979-56f4ed7aa63f.png)

```suggestion
        :class:`~optuna.pruners.PatientPruner`.
```

Reply:
Right, thank you. :pray: